### PR TITLE
Add composer.json file

### PR DESCRIPTION
**Pull request by schrapel**
Please add to packagist.org too

### CODE REVIEW

**Comment by schrapel** [comment]
I clearly have no idea how to Git

File: composer.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
+	"name": "schrapel/brightcove-video-connect",
```
Comment (sudar) [inline composer.json]: @schrapel Can you please change this to `brightcove/brightcove-video-connect`

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	"description": "A Brightcove™ Connector for WordPress that leverages enhanced APIs and Brightcove™ Capabilities",
+	"homepage": "https://github.com/10up/brightcove-video-connect/",
+	"keywords": [
+		"wordpress"
+	],
+	"support": {
+		"issues": "https://wordpress.org/support/plugin/brightcove-video-connect/"
+	},
+	"require": {
+		"php": ">=5.4.0",
```
Comment (sudar) [inline composer.json]: We support PHP 5.2.4+ and above (similar to [WordPress](https://wordpress.org/about/requirements/). Can you please change this as well?

File: composer.json (hunk 1 above)
Comment (schrapel) [inline composer.json]: Haha this reiterates that I have no idea how to use git. This was brightcove before. I've changed it

**Comment by sudar** [comment]
@schrapel 

Thanks for the pull request. I have merged it.

It will released as part of the next release.

**Pull request merged**